Makefile

```diff
@@ -10,17 +10,37 @@ SHELL := bash
 all: lint
 .PHONY: all
 
-lint: lint/helm lint/rules
+lint: build lint/helm lint/rules readme
+	@if ! git diff; then \
+		echo "Error: uncommitted changes."; \
+		exit 1; \
+	fi;
```

Comment:
`git diff` returns exit status 0 even if there are changes.

Reply:
Dang, I was using `--quiet` earlier and changed it, but my testing was obviously not extensive enough.

Reply:
Although I think bash is helping a bit here since CI is passing, so I think it works (by checking that stdout is empty), but I'll make it more explicit.